packages/react-sandpack/package.json

```
@@ -106,7 +106,7 @@
     "@types/react": "16.8.12",
     "@types/react-broadcast": "^0.6.1",
     "awesome-typescript-loader": "^3.5.0",
-    "babel-core": "7.0.0-bridge.0",
+    "@babel/core": "7.0.0",
```

I think it would be best to add a caret and redo `yarn`. That way we have the same version of `@babel/core` for all dependencies 😄 
```suggestion
    "@babel/core": "^7.0.0",
```

---

merged latest commits and updated PR